docs/source/supported_architectures.mdx

```diff
@@ -16,7 +16,36 @@ limitations under the License.
 
 # Supported architectures
 
-## Transformers
+## Training
+
+Training on AWS Trainium instances (Trn1) enables large-scale model training with distributed parallelism strategies.
+
+**Requirements:**
+- Model must be compatible with the Neuron SDK (if small enough to fit in 16GB, training is supported for any architecture that can be compiled)
+- **Memory constraint:** Each accelerator has 16GB memory for model weights, gradients, optimizer states, and activations
+- **For large models:** Custom modeling implementation with tensor parallelism and/or pipeline parallelism support is required
+
+The following architectures have custom modeling implementations with distributed training support:
+
+| Architecture             | Task            | Tensor Parallelism | Pipeline Parallelism | Expert Parallelism   |
+|--------------------------|-----------------|--------------------|----------------------|----------------------|
+| Llama, Llama 2, Llama 3  | text-generation | ✓                  | ✓                    | N/A                  |
+| Qwen3                    | text-generation | ✓                  | ✓                    | N/A                  |
+| Granite                  | text-generation | ✓                  | ✗                    | N/A                  |
+
+
+
+## Inference
+
+The following table lists the architectures and tasks that Optimum Neuron supports for inference on Amazon EC2 Inf2 instances. 
```

Comment:
And by the end of the day some large model inference is also recommended to be run on trn1...